File: app/Models/Category.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Category extends Model
{
    protected $fillable = [
        'name',
        'icon',
        'active',
    ];

    protected $casts = [
        'active' => 'boolean',
    ];

    public function tickets()
    {
        return $this->hasMany(Ticket::class);
    }
}

File: _CleanCity2_work/CleanCity2/app/src/main/java/com/example/cleancity/models/Category.java
package com.example.cleancity.models;

public class Category {
    private int id;
    private String name;
    private String icon;
    private boolean active;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getIcon() {
        return icon;
    }

    public boolean isActive() {
        return active;
    }

    @Override
    public String toString() {
        return name != null ? name : "Категория #" + id;
    }
}
